_posts/2016-01-13-OSS-Expectations.markdown

```diff
@@ -0,0 +1,89 @@
+
+### How did we start the process?
+
+We're lucky to have [technical](http://www.forbes.com/special-report/2014/30-under-30/art-and-style.html) [co-founders](https://www.linkedin.com/in/sebastiancwilich), and a CTO with a strong belief in [Open Source](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html). So we had a head-start, I respect that.
+
+Like all big ideas, it started out small. We'd abstract out shared concerns into libraries. This is something that anyone who has heard a little bit about Open Source can get behind. "We're building on top of _x_, so we really should give back out _x_+_y_" it's low-risk, and potentially high reward in hiring. Getting the company on-board with your OSS libraries is about acclimatisation.
+
+This works best by taking small incremental steps. You need buy-in from everyone involved though-out the process. Moving to OSS by Default is 30% technical work and 70% political relationships. There has to be infrastructure in place in order to not leak secrets, but you also need to ensure that the entire company doesn't feel threatened by the insight offered by opening the development process. There is no shortcut here.
+
+For me, this process involved talking with everyone involved in each project. Setting aside 1-on-1 time specifically on the subject to answer question about the ramifications for OSSing it. This ranged from "we will need to change the flow around _x_ and _y_." to "Yes, the competition will be able to see how we do _x_ and _y_." I came very prepared to these meetings.
+
+For our mobile apps, we progressively introduced Open by Default to our apps based on their age. We started out by creating a whole [new project](/blog/2014/11/13/eidolon-retrospective/) as Open Source. Then started applying what we had learned to [older](/blog/2015/04/28/how-we-open-sourced-eigen/) [projects](/blog/2015/08/06/open-sourcing-energy/). They had more risk, given that they were mature apps.
+
+### Couldn't someone make a business copying me?
+
+It's also worth remembering that an app being OSS does not stop it [being copied](http://venturebeat.com/2014/03/30/threes-vs-2048-when-rip-offs-do-better-than-the-original-game/). Or, well, your [entire business](http://www.bloomberg.com/bw/articles/2012-02-29/the-germany-website-copy-machine).
+
+Code only represents the _past to almost present_ of your business, losing a valuable colleague hurts because of their ability to move your business forwards. A "fresh replacement" has a while to go in terms of being able to make the change you want. Someone trying to build off your source, has to learn to understand your motivations, your aspirations and then try build what they'd like around that. It's not good business sense.
+
+### License Selection
+
+I covered this in [Licensing for OSS](/blog/2015/12/10/License-and-You/). If you want the TL:DR for Apps on app stores, jump to "Viral". We use MIT everywhere. It's worked out so far.
```

Comment:
Slightly dangerous, I’d urge for a bit more caution. The whole problem with licensing and infringement is that you cannot retroactively change it. So, because of our platform and the amount of time has undoubtedly gone into deciding on this very liberal license, we chose to use MIT, but that doesn’t mean to be a bit more paranoid if they’re possibly really out to get you.